en/get-started.md

```diff
@@ -14,27 +14,29 @@ This document explains how to create a simple Kubernetes cluster and use it to d
 
 These are the steps this document follows:
 
-1. [Create a Kubernetes cluster](#create-a-kubernetes-cluster)
+1. [Create a Kubernetes test cluster](#create-a-kubernetes-test-cluster)
 2. [Deploy TiDB Operator](#deploy-tidb-operator)
-3. [Deploy a TiDB cluster](#deploy-the-tidb-cluster)
+3. [Deploy a TiDB cluster and its monitoring services](#deploy-a-tidb-cluster-and-its-monitoring-services)
 4. [Connect to a TiDB cluster](#connect-to-tidb)
+5. [Upgrade a TiDB cluster](#upgrade-a-tidb-cluster)
+6. [Destroy a TiDB cluster](#destroy-a-tidb-cluster)
 
 If you have already created a Kubernetes cluster, you can skip to step 2, [Deploy TiDB Operator](#deploy-tidb-operator).
 
-If you want to do a production-quality deployment, use one of these resources:
+If you want to do a production-grade deployment, refer to one of these resources:
 
 + On public cloud:
     - [Deploy TiDB on AWS EKS](deploy-on-aws-eks.md)
     - [Deploy TiDB on GCP GKE (beta)](deploy-on-gcp-gke.md)
     - [Deploy TiDB on Alibaba Cloud ACK](deploy-on-alibaba-cloud.md)
 
-+ In an existing Kubernetes cluster:
-    1. Familiarize yourself with [Prerequisites for TiDB in Kubernetes](prerequisites.md)
-    2. Configure the local PV for your Kubernetes cluster to achieve low latency of local storage for TiKV according to [Local PV Configuration](configure-storage-class.md#local-pv-configuration)
-    3. Install TiDB Operator in a Kubernetes cluster according to [Deploy TiDB Operator in Kubernetes](deploy-tidb-operator.md)
-    4. Deploy your TiDB cluster according to [Deploy TiDB in General Kubernetes](deploy-on-general-kubernetes.md)
++ In a self-managed Kubernetes cluster:
+    - Familiarize yourself with [Prerequisites for TiDB in Kubernetes](prerequisites.md)
+    - [Configure the local PV](configure-storage-class.md#local-pv-configuration) for your Kubernetes cluster to achieve low latency for TiKV
```

Comment:
Doesn't this sentence need to be literally consistent with `参考[本地 PV 配置](configure-storage-class.md#本地-pv-配置)让 TiKV 使用高性能本地存储`?

Reply:
The original EN version was written by Kolbe and then translated to Chinese by another Chinese colleague, so I'm not sure which version is correct.
Let's leave it to @dragonly's discretion.

Reply:
I think "high performance" may be more accurate in this context, because local PV should achieve low latency and high throughput. What do you think @DanielZhangQD